CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/placeholder/PlaceholderUtilities.java

```diff
@@ -130,10 +130,10 @@ public static StoreGraph collapsePlaceholders(final StoreGraphRecordStore graph,
         // remove all transactions with type 'unknown' and all nodes with identifier 'unknown'
         if (cleanupGraph) {
             g.streamTransactions()
-                    .filter((t) -> t.getTypeValue().equals(SchemaTransactionTypeUtilities.getDefaultType()))
+                    .filter(t -> t.getTypeValue().equals(SchemaTransactionTypeUtilities.getDefaultType()))
```

Comment:
Could you expand the single character variable names here?

Reply:
Done